src/main/scala/beam/router/TravelTimeObserved.scala

```diff
@@ -166,6 +166,8 @@ object TravelTimeObserved extends LazyLogging {
   val histogramName: String = "simulation_vs_reference_histogram.png"
   val histogramBinSize: Int = 200
 
+  val MaxDistanceFromBeamTaz: Double = 500.0 // 500 meters
```

Comment:
@wrashid some time back we discussed this (it's about how far TAZ from provided data are from our TAZs) . Is it fine if it goes to develop?

Reply:
sure